.eslintrc

```diff
@@ -10,15 +10,15 @@
   "parserOptions": {
     "project": "./tsconfig.json"
   },
-  "plugins": [
-    "@typescript-eslint"
-  ],
+  "plugins": ["@typescript-eslint"],
   "rules": {
-    "@typescript-eslint/indent": "off"
+    "@typescript-eslint/indent": "off",
+    "@typescript-eslint/explicit-function-return-type": "off",
```

Comment:
I feel it is a bit verbose to have to declare all return types when them can inferred. The code keeps easier to read without return types in my opinion.

Reply:
👍